manifests/manager.pp

```diff
@@ -401,7 +401,8 @@
   # Install and configure Wazuh-manager package
 
   package { $wazuh::params_manager::server_package:
-    ensure  => $server_package_version, # lint:ignore:security_package_pinned_version
+    ensure          => $server_package_version, # lint:ignore:security_package_pinned_version
+    install_options => [ { '--enablerepo'  => 'wazuh' }, ],
```

Comment:
This parameter breaks the installation for Debian or Ubuntu systems:

E: The command line option --enablerepo=wazuh is not understood in combination with the other options

We need that this parameter is not added to these systems